itest/lnd_coop_close_external_delivery_test.go

```diff
@@ -0,0 +1,101 @@
+package itest
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+func testCoopCloseWithExternalDelivery(ht *lntest.HarnessTest) {
+	ht.Run("set at open", func(t *testing.T) {
+		tt := ht.Subtest(t)
+		testCoopCloseWithExternalDeliveryImpl(tt, true)
+	})
+	ht.Run("set at close", func(t *testing.T) {
+		tt := ht.Subtest(t)
+		testCoopCloseWithExternalDeliveryImpl(tt, false)
+	})
+}
+
+// testCoopCloseWithExternalDeliveryImpl ensurees that we have a valid settled
+// balance irrespective of whether the delivery address is in LND's wallet or
+// not. Some users set this value to be an address in a different wallet and
+// this should not affect our ability to accurately report the settled balance.
+func testCoopCloseWithExternalDeliveryImpl(ht *lntest.HarnessTest,
+	upfrontShutdown bool) {
+
+	alice, bob := ht.Alice, ht.Bob
+	ht.ConnectNodes(alice, bob)
+
+	// We need to give alice coins to open the channel, not sure why we
```

Comment:
So for me the test passes without funding Alice with extra coins, commented this line out and run the test, everything works fine

Reply:
It magically resolved itself on my machine too... 😒